CONTRIBUTING.md

```diff
@@ -211,7 +212,8 @@ git push --set-upstream origin feature/123/foo
 > Note:
 All PRs must include a commit message with a description of the changes made!
 
-Make sure you **sign off your commits**. To do this automatically check [this](https://github.com/keptn/lifecycle-toolkit/blob/main/CONTRIBUTING.md#auto-signoff-commit-messages).
+Make sure you **sign off your commits**.
+To do this automatically check [this](https://github.com/keptn/lifecycle-toolkit/blob/main/CONTRIBUTING.md#auto-signoff-commit-messages).
```

Comment:
Using "this" or "here" in cases like this is not a realy good practice.  Better to use the title of the document being referenced.

Reply:
i did not provide this content, i only added a break for better maintainability